Article 084: Turing Award

OmniaKahla opened the pull request:
## Attributes

<!-- Please use one of these items per category by removing the comment tag at the beginning and end of the line. -->

#### Length of text ####
<!-- :page_with_curl: short -->
:book:  long 

####  Difficulty to read ####
 :mouse:  light 
<!-- :elephant:  heavy -->

####  Difficulty to understand #### 
 :school_satchel:  easy 
<!-- :mortar_board:  difficult -->

####  Type of text #### 
<!-- :laughing:  funny -->
:bowtie:  informative 
<!-- :sunglasses:  catchy -->

####  Type of header #### 
<!-- :santa:  funny --> 
<!-- :bulb:  informative -->
:boom:  catchy

Fjolyo commented:
## Plenum Suggestions

- Maybe mention current winner
- First paragraph seems negative
- Maybe use optional text as first paragraph
- Mention "more interested in..." instead of "not interested in..."
- Reference touring machine as the "father of computer science" and leave out the other awards

OmniaKahla commented:
All changes have been done as follows:
- Added the first and last winner. 
- Added the optional text to the first paragraph.
- Updated more interested instead of not interested in
- Added in the text that the Turing award is the Nobel prize for computer scientists.

marcelluethi commented:
@noverney 
@OmniaKahla 
@vogti 
Unfortunately I forgot to set the grammar label on this article and to ask Normand for a review. 
Maybe it is still possible to do the language review for this article and to make the article fit for the webpage?

I am sorry for the inconvenience.

noverney commented on articles/article-84-turing-award.md:
```diff
@@ -0,0 +1,28 @@
+<!-- BEGIN TITLE -->
+# Why no Nobel prize for computer scienctists?
```
spelling -> computer scientists

noverney replied:
Why is there no Nobel prize for computer scientists?

noverney commented on articles/article-84-turing-award.md:
```diff
@@ -0,0 +1,28 @@
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Some people say it is because that Alfred Nobel’s fiancée had an affair with a mathematician, others say because he was more interested in Physics, Chemistry, Medicine, Literature, and Peace than in Mathematics, and computer science is a part of the Mathematics field. 
```
Some say it was because Alfred Nobel's fiancé had an affair with a mathematician, which can be considered the foundation of computer science, while others believe Nobel was just more interested in physics, chemistry, medicine, literature and peace than in mathematics. 
- Alfred Nobel also lived in the 1800s, so the idea he did not like computer science is absurd. The earliest known computer was known as the Analytical machine, which was a glorified calculator. The first programmable computers did not become relevant until the 1900s. That is why I rephrased it in such a way that the focus was on mathematics.

noverney commented on articles/article-84-turing-award.md:
```diff
@@ -0,0 +1,28 @@
+<!-- BEGIN BODY -->
+Some people say it is because that Alfred Nobel’s fiancée had an affair with a mathematician, others say because he was more interested in Physics, Chemistry, Medicine, Literature, and Peace than in Mathematics, and computer science is a part of the Mathematics field. 
+
+Luckily, there is a "Nobel Prize of Computing" which is the A.M. Turing Award. It is an annual prize given by the Association for Computing Machinery (ACM) to an individual who makes (a) major contribution(s) to the computer field. The award is named after Alan Mathison Turing, who was a mathematician and computer scientist. The first award was at 1966 and went to Alan J. Perlis for his achieveme ...
```
- you don't need the parenthesis at all since this is not a legal document and the plural case covers the singular. So just say .. who makes major contributions to the computer field. 
- In 1966, Alan J. Perlis received the first Turing award for his chievements in advanced computer programming techniques and compiler construction. 
- technically the last one did not get sent out yet since the world has not yet ended (as far as I can tell). Most recently, in 2018, Yoshua Bengio, Geoffrey Hintonor, and Yann LeCun received the award for improving deep neural networks. (the issue with this sentence is it gets frequently outdated)

OmniaKahla commented:
All changes are done.
